src/go/rpk/pkg/api/admin/api_broker.go

```diff
@@ -41,9 +41,10 @@ type Broker struct {
 // Brokers queries one of the client's hosts and returns the list of brokers.
 func (a *AdminAPI) Brokers() ([]Broker, error) {
 	var bs []Broker
-	defer func() {
+	sort := func() {
```

Comment:
Which lint is causing this change? I don't think this buys anything.

Reply:
Because revive's defer takes the less function as a return of the deferred function.